Makefile

```diff
@@ -0,0 +1,7 @@
+# Makefile executes first target if all is not specified
+init:
+	pip install pipenv --upgrade
+	pipenv install --dev --skip-lock
+
+test:
+	scripts/ci
```

Comment:
I'd rather we stick with the `/scripts/install` style than introduce a `Makefile`.

Reply:
Fair! 👍